src/completions/generate-v1.ts

```diff
@@ -0,0 +1,277 @@
+import * as fs from 'fs-extra';
+import * as path from 'path';
+
+import { checkInstalledVersion } from '../updates/appc/core';
+import { CustomError, getAlloyCompletionsFileName, getAlloyVersion, getSDKCompletionsFileName } from './util';
+
+import { homedir } from 'os';
+
+const completionsVersion = 1;
```

Comment:
Please use the enum from the main file for this and the other version references in this file